CHANGES.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## v0.5.0
+
+Users willing to use Pmac or NDSSCPimega should update to this release. Users
+interested in fully static linked binaries through musl build should update as
+well.
+
```

Comment:
I don't think we should mention the musl build this way, because it was a use case we didn't cover, so updating to it doesn't make sense, IMO.

"A musl-based image has been added, enabling support for linking fully static binaries for deployment outside of containers."

Reply:
Nice. That way it is clearer indeed what this is for. Thanks!